config/transform-elasticsearch/documents/datadiscovery_study.json

```diff
@@ -41,14 +41,57 @@
         }
       ]
     },
+    "entryType": {
+      "{list}": [
+        {
+          "{or}": [
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Genotyping", "Genotyping Study",
+                  "allele size", "genotype"
+                ]
+              },
+              "{with}": {
+                "replaced_by": "Genotyping Study"
+              }
+            },
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Phenotypes", "Phenotyping", "Field Experiement",
```

Comment:
Typo: `Experiement`

Reply:
Those typo reflect the type of entry type there are in the different brapi endpoint. Therefore we have to leave them as they are.